packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx

```diff
@@ -73,6 +73,42 @@ interface Props {
     payload: MetadataAddPayload;
 }
 
+const getLocalizedActions = (type: MetadataAddPayload['type']) => {
+    const defaultMessages = {
+        add: <Translation id="TR_LABELING_EDIT_LABEL" />,
```

Comment:
Looks like this ID is not correct (and the other ones below for "add" actions too). Looking at the messages file, there are separate add messages.

Reply:
ehh I missed that. thanks!